lib/search/presenters/result_presenter.rb

```diff
@@ -87,9 +87,14 @@ def add_debug_values(result)
       # debugging is requested, so it's nicer to be explicit about what score
       # it is.
       result[:es_score] = raw_result["_score"]
+      # LearnToRank generated values
+      result[:model_score] = raw_result["model_score"]
+      result[:original_rank] = raw_result["original_rank"]
+      result[:combined_score] = raw_result["combined_score"]
+
       result[:_id] = raw_result["_id"]
 
-      if raw_result["_explanation"]
+      if raw_result["_explanation"] && search_params.debug[:explain]
```

Comment:
Why is this `&&` bit needed?

Reply:
We don't want to return explanation to end users unless they've requested them in their search params. However, the model currently requires the scores within the explanation, so we always need to extract it from ES (potential performance problem there to address).